client/me/account-close/main.jsx

```diff
@@ -0,0 +1,126 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import page from 'page';
+import Gridicon from 'gridicons';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import HeaderCake from 'components/header-cake';
+import ActionPanel from 'components/action-panel';
+import ActionPanelTitle from 'components/action-panel/title';
+import ActionPanelBody from 'components/action-panel/body';
+import ActionPanelFigure from 'components/action-panel/figure';
+import ActionPanelFigureHeader from 'components/action-panel/figure-header';
+import ActionPanelFigureList from 'components/action-panel/figure-list';
+import ActionPanelFigureListItem from 'components/action-panel/figure-list-item';
+import ActionPanelLink from 'components/action-panel/link';
+import ActionPanelFooter from 'components/action-panel/footer';
+import Button from 'components/button';
+
+class AccountSettingsClose extends Component {
+	goBack = () => {
+		page( '/me/account' );
+	};
+
+	handleDeleteClick = event => {
+		event.preventDefault();
+	};
+
+	render() {
+		const { translate } = this.props;
+		const hasAtomicSites = false;
+
+		return (
+			<div className="account-close main" role="main">
+				<HeaderCake onClick={ this.goBack }>
+					<h1>{ translate( 'Close account' ) }</h1>
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Close Account' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).